chall04/alcohen.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+
+# Version: Python 3.6.9
+
+def error_input(msg):
+    print(msg)
+    quit()
```

Comment:
instead of creating your own function `error_input` you could use `sys.exit("your message")`